### PR TITLE
Add pre-commit hook for pylint with specific rules

### LINKED ISSUE #431: [Shell] Add pre commit hook
🚨 **Attention: Please read the guidelines before applying; otherwise, you will be ignored.** 🚫📋
0. Please, join the telegram group to keep in touch https://t.me/+m0IGRiBvLtsyMzIy 
1. Read guideline [here](https://gist.github.com/djeck1432/f722f5fe518c80b248ff609627da25e5)
2. Add to the project pre-commit hook, which will run before each commit
3. Add pylint checker for it pylint rules which we use: `pylint $(git ls-files '*.py') --disable=all --enable=C0114,C0115,C0116,C0301`

### PR DESCRIPTION
**Pull request by 1nonlypiece**
closes #431

### CODE REVIEW

**Comment by 1nonlypiece** [comment]
@iamnovichek  I've added the pre-commit configuration file, but the tool isn't installed yet, and the pre-commit install command still needs to be run to apply it. Should this be part of the task?

**Comment by 1nonlypiece** [comment]
Should this setup be handled using Poetry and Docker? If so, we could limit pre-commit usage to Docker containers, but most contributors will likely commit from their local machines. What are your suggestions @iamnovichek ?

**Comment by iamnovichek** [comment]
> Should this be part of the task?

@1nonlypiece yeah, please do it as well

**Comment by iamnovichek** [comment]
> Should this setup be handled using Poetry and Docker? If so, we could limit pre-commit usage to Docker containers, but most contributors will likely commit from their local machines. What are your suggestions @iamnovichek ?

I think we will require that each contributor runs `pre-commit install` command on pull our repository. So install it locally pls as well and add to `pyproject.toml`

**Comment by 1nonlypiece** [comment]
@iamnovichek I have added it in `pyproject.toml` and wondering how to install it locally?

**Comment by 1nonlypiece** [comment]
Can we just document to install `pre-commit` locally & run `pre-commit install` command?

**Comment by iamnovichek** [comment]
> Can we just document to install `pre-commit` locally & run `pre-commit install` command?

Yep, that's exactly what I meant

**Comment by 1nonlypiece** [comment]
Updated! @iamnovichek

**Comment by 1nonlypiece** [comment]
This PR is ready for review @iamnovichek

File: README.md [hunk 1]
```diff
@@ -248,3 +248,16 @@ Run command to create migration file
...
 alembic -c web_app/alembic.ini revision --autogenerate -m "migration message"
 ```
+
+## Pre-commit Setup
+
+To ensure code quality, install pre-commit hooks locally:
+
+1. Install pre-commit:
+   ```bash
+   pip install pre-commit
```
Comment (iamnovichek) [inline README.md:258]: Pls add 3 into `pip install pre-commit` to be `pip3 install pre-commit`

**Review by iamnovichek — CHANGES_REQUESTED** [review]

**Pull request merged**